mobile/src/people/departments/departments-horizontal-scrollable-list.tsx

```diff
@@ -85,11 +155,9 @@ export class DepartmentsHScrollableList extends Component<DepartmentsHScrollable
                 this.props.requestEmployeesForDepartment(this.props.headDepartment.departmentId);
                 const stubDN: DepartmentsTreeNode = {
                     departmentAbbreviation: null,
-                    departmentChiefId: this.props.headDepartment.head.employeeId,
-                    head: this.props.headDepartment.head,
+                    departmentChiefId: this.props.headDepartment.departmentChiefId,
                     parent: null,
                     children: null,
-                    subordinates: null,
                     departmentId: 'subordinates'
```

Comment:
Move 'subordinates' to const. I've seen it somewhere else

Reply:
addressed